### PR TITLE
Add one click installer figures to assets & hosted on readthedocs

### PR DESCRIPTION
**Pull request by JinBridger**

### CODE REVIEW

File: readthedocs/source/index.rst [hunk 1]
```diff
@@ -100,3 +100,47 @@ Image address:
 .. code-block::
 
   https://llm-assets.readthedocs.io/en/latest/_images/bigdl_logo.png
+
+
+one-click-installer-screen-capture.gif
+-------------------------
```
Comment (Oscilloscope98) [inline readthedocs/source/index.rst:106]: Make this ---- longer than the title to avoid syntax warning

**Comment by jason-dai** [comment]
Create a subfolder in `assets` for these pictures

**Comment by JinBridger** [comment]
> @JinBridger Note that even if the image is in a subfolder under assets, the image link is still `https://llm-assets.readthedocs.io/en/latest/_images/新的图片名`

Got it! By the way I've replaced the screen capture gif. If there's any other problem, feel free to tell me.

**Review by Oscilloscope98 — APPROVED** [review]
LGTM

**Pull request merged**